web/app/js/components/Top.jsx

```diff
@@ -144,13 +144,15 @@ class Top extends React.Component {
     });
   }
 
-  updateTapClosingState(isTapClosing) {
+  updateTapClosingState(isTapClosing, isTapRequestInProgress) {
     this.setState({
-      tapIsClosing: isTapClosing
+      tapIsClosing: isTapClosing,
+      tapRequestInProgress: isTapRequestInProgress
     });
   }
 
   render() {
+
```

Comment:
You added an extra space here, which you don't need if you are returning right below the `render()`. See [React Component docs code example](https://reactjs.org/docs/react-component.html)